### PR TITLE
:sparkles: feat: SQ-65952 [front] Criar novo componente de slider

### PR DESCRIPTION
**Pull request by wandersonsales-dev**
Card: [SQ-65952](https://duopana.myjetbrains.com/youtrack/issue/SQ-65952/front-Criar-novo-componente-de-slider) 

## Summary by CodeRabbit

- **New Features**
	- Introduced a "Flow" component in the documentation with a responsive grid layout and visual examples.
	- Added styling for the "Flow" component, enhancing user experience with interactive list items and visual feedback.
  
- **Chores**
	- Updated import statements for consistency in the SCSS files.

### CODE REVIEW

**Comment by htamagnus** [comment]
Tem algumas sugestões do code rabbit que vale a pena ver se vale considerar @wandersonsales-dev

**Review by htamagnus — COMMENTED** [review]
Tem algumas sugestões do code rabbit que vale a pena ver se vale considerar @wandersonsales-dev

File: src/scss/components/_flow.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      border-radius: 20px;
+      background-color: var(--color_border_box_neutral_primary);
+      flex-shrink: 0;
+      min-width: 12px;
+
+      &.active {
+        background: var(--primary_color);
+        width: -webkit-fill-available;
+        border-radius: 20px;
+        flex-shrink: 1;
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/scss/components/_flow.scss:31]: Done :rocket:

**Comment by wandersonsales-dev** [comment]
> Tem algumas sugestões do code rabbit que vale a pena ver se vale considerar @wandersonsales-dev

Feito 🚀

**Review by htamagnus — APPROVED** [review]

**Comment by wandersonsales-dev** [comment]
@JonasPeres code review

**Review by JonasPeres — APPROVED** [review]

**Pull request merged**